parking_permits_app/constants.py

```diff
@@ -62,6 +62,12 @@ class ParkingPermitStatus(Enum):
     EXPIRED = _("EXPIRED")
 
 
+class RefundStatus(Enum):
```

Comment:
Not blocking the merge, but some minor notes. This is not the right way to define enums to be used for model choices. The Enum value should not be translatable texts, which are dynamic values depending on the active language. The label for the choice should be defined separately, but not as Enum values. I think we need to change all these Enums at some point.